pvlib/temperature.py

```diff
@@ -186,6 +187,72 @@ def sapm_module(poa_global, temp_air, wind_speed, a, b):
     return poa_global * np.exp(a + b * wind_speed) + temp_air
 
 
+def sapm_cell_from_module(module_temperature, poa_global, deltaT,
+                          irrad_ref=1000):
+    r'''
+    Calculate cell temperature from module temperature using the Sandia Array
+    Performance Model.
+
+    See [1]_ for details on the Sandia Array Performance Model.
+
+    Parameters
+    ----------
+    module_temperature : numeric
+        Temperature of back of module surface [C].
+
+    poa_global : numeric
+        Total incident irradiance [W/m^2].
+
+    deltaT : float
+        Parameter :math:`\Delta T` in :eq:`sapm2` [C].
+
+    irrad_ref : float, default 1000
+        Reference irradiance, parameter :math:`E_{0}` in
+        :eq:`sapm2` [W/m^2].
+
+    Returns
+    -------
+    numeric, values in degrees C.
+
+    Notes
+    -----
+    The model for cell temperature :math:`T_{C}` is given by Eq. 12 in [1]_.
+
+    .. math::
+       :label: sapm2
+
+       T_{C} = T_{m} + \frac{E}{E_{0}} \Delta T
+
+    The module back surface temperature :math:`T_{m}` is implemented in
+    :py:func:`~pvlib.temperature.sapm_module`.
+
+    Model parameters depend both on
+    the module construction and its mounting. Parameter sets are provided in
+    [1]_ for representative modules and mounting, and are coded for convenience
+    in ``pvlib.temperature.TEMPERATURE_MODEL_PARAMETERS``.
```

Comment:
The constant dictionary `TEMPERATURE_MODEL_PARAMETERS` is not included in the public api (`api.rst`) even though the documentation references it numerous places. If it's worth mentioning to users then IMO I think it should be published. Perhaps a separate bug/issue?